strongback/src/org/strongback/hardware/HardwareTalonSRX.java

```diff
@@ -454,7 +455,7 @@ public TalonSRX setForwardSoftLimit(int forwardLimitDegrees) {
         // Compute the desired forward limit in terms of the current selected input sensor ...
         if ( this.selectedInput != null ) {
             double rawPosition = this.selectedInput.rawPositionForAngleInDegrees(forwardLimitDegrees);
-            talon.setForwardSoftLimit(rawPosition);
+            talon.setForwardSoftLimit((int) rawPosition);
```

Comment:
Do you know what number `rawPosition` is supposed to be? Do we have to scale it, `ceil` it, etc?